tests/device_tests/test_session_id_and_passphrase.py

```diff
@@ -375,6 +376,40 @@ def call(passphrase: str, expected_result: bool):
     call(passphrase="A" * 49 + "š", expected_result=False)
 
 
+@pytest.mark.setup_client(passphrase=True)
+def test_hide_passphrase_from_host(client: Client):
```

Comment:
pretty sure T1 is going to fail on this. we don't want the feature for T1 so please add `skip_t1`

Reply:
Right, added in [c2cd56a](https://github.com/trezor/trezor-firmware/pull/2829/commits/c2cd56a68a607224e2d392b32d60c2bc7481cb9d)